.github/workflows/deploy-to-eks.yaml

```diff
@@ -2,21 +2,29 @@ name: Create EKS cluster, deploy CKF and run bundle test
 on:
   workflow_dispatch: # This event allows manual triggering from the Github UI
     inputs:
+      bundle_version:
+        description: 'Comma-separated list of bundle versions e.g. "1.7","1.8"'
+        default: '"1.8"'
```

Comment:
Maybe the default can be latest, current and last release, that way we can avoid having a bunch of versions all over the workflow in the next lines.

Also, maybe we could have some sort of env variable that we can use everywhere, so when we need to update this workflow, we don't have to look everywhere. Maybe something like:

```yaml
CURRENT_VERSION: 1.8
PREVIOUS_VERSION : 1.7
LATEST: latest

# Then you could use it like

bundle-version: ${{ fromJSON(format('[{0}]', inputs.bundle_version || env.PREVIOUS_VERSION, etc.)) }}
```

Reply:
Yikes, okay, I think then the defaults will help here anyway.